charts/tidb-cluster/templates/_helpers.tpl

```diff
@@ -37,6 +37,13 @@ config-file: |-
     {{- if .Values.pd.config }}
 {{ .Values.pd.config | indent 2 }}
     {{- end -}}
+    {{- if .Values.enableTLSCluster }}
+  [security]
+  cacert-path = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+  cert-path = "/var/lib/pd-tls/cert"
+  key-path = "/var/lib/pd-tls/key"
+    {{- end -}}
+
```

Comment:
refer https://github.com/pingcap/tidb-operator/pull/750#discussion_r317957297

Reply:
Adding `security` config section here is fragile, users may provide this section in the `.Values.pd.config` too thus causing invalid configuration file.